tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,186 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { access, readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import fetch from 'node-fetch';
+import { EpicGeneratorSchema } from './schema';
+
+function validateSchema(schema: EpicGeneratorSchema) {
+  if (!schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository for the issue');
```

Comment:
```suggestion
    throw new Error('Must provide a valid repository in the format {owner}/{repositoryName}');
```

nit: is this better ?

Reply:
Added this and improved the other error message too.